coordinator/package.json

```diff
@@ -9,7 +9,7 @@
     "start": "node coordinator/src/app.js --use_strict",
     "start-with-env": ". ../../scripts/set_env_vars.sh && node coordinator/src/app.js --use_strict",
     "start-dev": "ts-node-dev --respawn --transpileOnly src/app.ts",
-    "test": "jest --reporters=jest-junit",
```

Comment:
reminder to add this back. this is used by SA

Reply:
reverted, we might want to chat about having two separate test commands, one with the reporting and one without for local testing

Reply:
spoke to Dan about that, could go with that option, or you could use this command to cover both:

`"test": "jest --reporters=default --reporters=jest-junit"`